astropy_helpers/extern/numpydoc/plot_directive.py

```diff
@@ -0,0 +1,642 @@
+"""
```

Comment:
Ok I removed this and four other files removed in that PR in 026a53a584b30cb5717986cb2580201bd668d9d3